### PR TITLE
[HOPSWORKS-1809] Remove deprecated Python management functionality

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing-ee/pull/99

File: Berksfile [hunk 1]
```diff
@@ -3,21 +3,21 @@ source 'https://supermarket.chef.io'
 
 metadata
 
-cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
+cookbook 'conda', github: "robzor92/conda-chef", branch: "HOPSWORKS-1809"
```
Comment (SirOibaf) [inline Berksfile:6]: Make sure you revert these changes before merging.

File: templates/default/sql/dml/1.4.0.sql.erb [hunk 2]
```diff
@@ -5,4 +5,5 @@ REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("jupyter_group", "<
 -- the path is hardcoded because it is the path inside the docker image, not on the machine.
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("anaconda_dir", "/srv/hops/anaconda");
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_name", "<%= @registry_host %>");
-REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
\ No newline at end of file
+REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
+DELETE FROM `hopsworks`.`variables` WHERE `id`='provided_python_lib_names';
```
Comment (SirOibaf) [inline templates/default/sql/dml/1.4.0.sql.erb:9]: why did you remove this one?

File: files/default/sql/ddl/updates/1.4.0.sql [hunk 3]
```diff
@@ -21,3 +21,7 @@ UPDATE `hopsworks`.`jupyter_settings` `j`
 JOIN `hopsworks`.`project` `p`
 ON `j`.`project_id` = `p`.`id`
 SET `j`.`base_dir` = CONCAT('/Projects/',`p`.`projectname`,'/Jupyter');
+
+ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN machine_type;
+ALTER TABLE `hopsworks`.`python_dep` DROP COLUMN machine_type;
+ALTER TABLE `hopsworks`.`hosts` DROP COLUMN conda_enabled;
```
Comment (SirOibaf) [inline files/default/sql/ddl/updates/1.4.0.sql:27]: in `recipe/default.rb` we still have some code related to conda/non-conda hosts. could you please remove it as well?

Comment (robzor92) [reply files/default/sql/ddl/updates/1.4.0.sql:27]: We are using those variables in older dml files. I don't know what the correct approach for purging that code is. https://github.com/logicalclocks/hopsworks-chef/blob/49f7e7c1e54bb298ea2993e67d4f589456ce2e3b/recipes/default.rb#L308

File: templates/default/sql/dml/1.4.0.sql.erb (hunk 2 above)
Comment (robzor92) [reply templates/default/sql/dml/1.4.0.sql.erb:9]: It is not needed anymore. Now we list all libraries in the environment so this variable is useless.

File: Berksfile (hunk 1 above)
Comment (robzor92) [reply Berksfile:6]: Sure
